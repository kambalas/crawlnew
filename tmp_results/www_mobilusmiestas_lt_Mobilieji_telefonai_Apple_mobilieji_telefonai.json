[
  {
    "name": "Apple iPad 10.9 2022 10 gen. 256GB Wifi",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPad 10.9 2022 10 gen. 256GB Wifi",
    "price": "557,00",
    "error": false
  },
  {
    "name": "Apple iPad 10.9 2022 10 gen. 64GB Wifi",
    "price": "345,00",
    "error": false
  },
  {
    "name": "Apple iPad AIR 2024 11 128GB 6th gen",
    "price": "683,00",
    "error": false
  },
  {
    "name": "Apple iPad Pro 11 128GB WiFi (2022 Model)",
    "price": "977,00",
    "error": false
  },
  {
    "name": "Apple iPhone 12 256GB",
    "price": "818,00",
    "error": false
  },
  {
    "name": "Apple Iphone 14 128GB",
    "price": "649,00",
    "error": false
  },
  {
    "name": "Apple Iphone 14 Plus 128GB",
    "price": "759,00",
    "error": false
  },
  {
    "name": "Apple Iphone 14 Plus 256GB",
    "price": "1.049,00",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro Max 128GB",
    "price": "1.298,00",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro Max 256GB",
    "price": "1.313,00",
    "error": false
  },
  {
    "name": "Apple Iphone 15 128GB",
    "price": "732,00",
    "error": false
  },
  {
    "name": "Apple iPhone 15 Pro Max 256GB",
    "price": "1.287,00",
    "error": false
  },
  {
    "name": "Apple iPad 10.9 2022 10 gen. 256GB Wifi",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPad 10.9 2022 10 gen. 64GB Wifi",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPad AIR 2024 11 128GB 6th gen",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPad Pro 11 128GB WiFi (2022 Model)",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 12 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple Iphone 14 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple Iphone 14 Plus 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple Iphone 14 Plus 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro Max 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro Max 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple Iphone 15 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 15 Pro Max 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 15 Pro Max 512GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple Iphone 16 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple Iphone 16 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro Max 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro Max 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPad 10.9 2022 10 gen. 256GB Wifi",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPad 10.9 2022 10 gen. 64GB Wifi",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPad AIR 2024 11 128GB 6th gen",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPad Pro 11 128GB WiFi (2022 Model)",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 12 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple Iphone 14 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple Iphone 14 Plus 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple Iphone 14 Plus 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro Max 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro Max 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple Iphone 15 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 15 Pro Max 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 15 Pro Max 512GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple Iphone 16 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple Iphone 16 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro Max 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 256GB",
    "price": "",
    "error": false
  }
]