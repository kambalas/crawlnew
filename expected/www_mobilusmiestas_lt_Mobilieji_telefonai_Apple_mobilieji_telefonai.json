[
  {
    "name": "Apple iPhone 12 256GB",
    "price": "818,00 €",
    "error": false
  },
  {
    "name": "Apple iPhone 14 128GB",
    "price": "649,00 €",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Plus 128GB",
    "price": "759,00 €",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Plus 256GB",
    "price": "1.049,00 €",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro Max 128GB",
    "price": "1.298,00 €",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro Max 256GB",
    "price": "1.313,00 €",
    "error": false
  },
  {
    "name": "Apple iPhone 15 128GB",
    "price": "732,00 €",
    "error": false
  },
  {
    "name": "Apple iPhone 15 Pro Max 256GB",
    "price": "1.287,00 €",
    "error": false
  },
  {
    "name": "Apple Iphone 14 Plus 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro Max 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro Max 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple Iphone 15 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 15 Pro Max 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 15 Pro Max 512GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple Iphone 16 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple Iphone 16 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro Max 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 12 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple Iphone 14 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple Iphone 14 Plus 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple Iphone 14 Plus 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro Max 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro Max 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple Iphone 15 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 15 Pro Max 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 15 Pro Max 512GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple Iphone 16 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple Iphone 16 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro Max 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 128GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro 256GB",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 15",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 15 Plus",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 15 Pro",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 15 Pro Max",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Plus",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 16 Pro Max",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Pro Max",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 14 Plus",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 13 Pro",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 13",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 13 Pro Max",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 13 mini",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 12/12 Pro",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 12 Pro Max",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 12 Mini",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 11 Pro Max",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 11 Pro",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone 11",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone XR",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone Xs Max",
    "price": "",
    "error": false
  },
  {
    "name": "Apple iPhone X/Xs",
    "price": "",
    "error": false
  }
]